### ISSUE TITLE
You will still update to XE8?

### ISSUE THREAD

**Issue by humbertocastelo** [hgourvest/dcef3#2]
You will still update to XE8?

**Comment by hgourvest** [comment]
I guess your question is related to Firemonkey, Firemonkey component do not compile in XE>2, VCL components should just works.

**Comment by humbertocastelo** [comment]
Yes I understood. Once I asked the question I performed some tests and is working and well updated, thank you, your repository is the most updated I found on the internet (2378). Is possible to use it in Lazarus, you have any plans to convert? I saw other repositories but are outdated.
